grammars/clojure.cson

```diff
@@ -206,10 +325,20 @@
         'name': 'constant.character.escape.clojure'
       }
     ]
+  'namespace-symbol':
+    'patterns': [
+      { # copied from #symbol, plus a / at the end. Matches the "app/" part of
+        # "app/*config*"
+        'match': '([\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*][\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*\\d]+)/'
+        'captures':
+          '1':
+            'name': 'meta.symbol.namespace.clojure'
+      }
+    ]
   'symbol':
     'patterns': [
       {
-        'match': '(\\w[\\w\\d]+)'
+        'match': '([\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*][\\w\\.\\-\\_\\:\\+\\=\\>\\<\\!\\?\\*\\d]+)'
```

Comment:
I don't think you need to have these all escaped since they are already in a character class.  The only ones for sure would be things like `\w` and `-` to differentiate it from a range.